src/js/common/schemaform/review/SubmitButtons.jsx

```diff
@@ -49,6 +51,21 @@ export default function SubmitButtons(props) {
         </div>
       </div>
     );
+  } else if (submission.status === 'throttledError') {
+    submitButton = (
+      <ProgressButton
+        onButtonClick={onSubmit}
+        buttonText="Submit Application"
+        buttonClass="usa-button-primary"/>
+    );
+    submitMessage = (
+      <div className="usa-alert usa-alert-error schemaform-failure-alert">
+        <div className="usa-alert-body">
+          <p className="schemaform-warning-header"><strong>We’ve run into a problem</strong></p>
+          <p>We’re sorry. Your submission didn’t go through because we received too many requests from you. Please wait {timeFromNow(moment.unix(submission.extra))} and submit your request again.</p>
```

Comment:
Should this match the message above?
"You’ve reached the limit for the number of submissions we can accept at this time. Please try again in..."
Or vice versa?

Reply:
They're different messages in the linked issue, though @peggygannon  can confirm.